py-polars/tests/unit/datatypes/test_enum.py

```diff
@@ -402,3 +394,27 @@ def test_enum_cast_from_other_integer_dtype_oob() -> None:
         pl.ComputeError, match="conversion from `u64` to `u32` failed in column"
     ):
         series.cast(enum_dtype)
+
+
+def test_enum_creating_col_expr() -> None:
```

Comment:
If I am not mistaken, this already runs on main without any of the other changes.

Reply:
Because we should be able to convert the python class object to the rust datatype without hte need for `None` in the constructor